api/migrations/0017_remove_profile_unique_code.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 2.0.5 on 2018-07-29 21:57
+
+from django.db import migrations
```

Comment:
Instead of removing the migration this way
You shouldn't be having any migration at all